src/js/wrapper.ts

```diff
@@ -642,6 +644,17 @@ export const NATIVE: SentryNativeWrapper = {
     return RNSentry.getCurrentReplayId() || null;
   },
 
+  async crashedLastRun(): Promise<boolean | null> {
+    if (!this.enableNative) {
+      return null;
+    }
+    if (!this._isModuleLoaded(RNSentry)) {
+      return null;
+    }
+
+    return RNSentry.crashedLastRun();
```

Comment:
Related to the native type comment, ensuring only null is passed back if the bool is not evaluated.

```suggestion
    const result = RNSentry.crashedLastRun();
    return typeof result === 'boolean' ? result : null;
```